packages/contracts/contracts/DependencyRegistryStorageLib.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import "./interfaces/v0.8.x/IAdminACLV0.sol";
+import "@openzeppelin-4.7/contracts/utils/structs/EnumerableSet.sol";
+
+library DependencyRegistryStorageLib {
```

Comment:
nit: add natspec comments to library, structs, and functions, including internal functions in libraries. See `SetPriceLib.sol` for example natspec